### DISCUSSION TITLE
Location wise filter

### DISCUSSION THREAD

**Discussion by myselfpankajk** [inspireui/support#5228] [New Feature/Ideas Requests] [answered]
Hello Sir, please add location wise stores and products filter just after app launch ...
App automatically detect location and shows only nearby product and Stores.

**Comment by pmcjamal** [comment, marked as answer]
yes, the Mandatory feature
I have multiple stores and some stores deliver only in some cities, not all cities

how can I show to the customer only the product which is available to deliver to his delivery address?
Currently showing all the products to all cities irrespective of delivery available or not

**Comment by faam155** [comment]
+1

**Comment by devahmad** [comment]
+1

**Comment by pasarsegar** [comment]
+1
